vendor/k8s.io/kubernetes/pkg/api/types.go

```diff
@@ -1256,6 +1256,9 @@ const (
 // Capability represent POSIX capabilities type
 type Capability string
 
+// CapabilityAll represent all POSIX capabilities types.
+var CapabilityAll Capability = "*"
```

Comment:
There should be changes in here and v1 types to note on the description that `*` is an available wildcard